backend/src/main/java/com/example/subjecthub/dto/AddCommentRequest.java

```diff
@@ -0,0 +1,33 @@
+package com.example.subjecthub.dto;
+
+/**
+ * DTO sent by user when adding a comment.
+ */
+public class AddCommentRequest {
+    private Long userId;
```

Comment:
As mentioned earlier, the user is attached implicitly during the request, no need to have the user id explicitly present here. I think this request can simply be consist of a String field.

Reply:
fixed